config/barong.yml

```diff
@@ -16,3 +16,6 @@ document_types:
   - Utility Bill
   - Residental
   - Institutional
+data_storage_titles:
```

Comment:
I would name it `user_storage_titles`

Reply:
about `user_storage` dont you think its a bit unclear?
user_storage sounds like a storage of users?

Reply:
user_personal_storage ?